pandas/tseries/tests/test_resample.py

```diff
@@ -619,6 +621,47 @@ def test_resample_interpolate(self):
             df.resample('1T').asfreq().interpolate(),
             df.resample('1T').interpolate())
 
+    def test_resample_empty_series(self):
+        # GH12771 & GH12868
+
+        s = self.create_series()[:0]
+
+        for freq in ['M', 'D', 'H']:
+            methods = resample_methods.copy()
+            # need to test for ohlc from GH13083
+            methods.remove('ohlc')
+            for method in methods:
+                expected_index = s.index
+
+                result = getattr(s.resample(freq), method)()
+                expected = s
+                assert_index_equal(result.index, expected_index)
+                if method == 'size':
+                    # GH12871
+                    assert_series_equal(result, expected, check_dtype=False,
```

Comment:
put a TODO here (I want this to break when this is fixed, so we are KNOW to change this test). The way this is written it still would work. Maybe assert that the ``result.name is None``. This is only for ``size``.